classes/Village.php

```diff
@@ -53,4 +64,39 @@ public static function getLocation(System $system, string $village_name): ?Trave
 
         return null;
     }
+    public static function repName($rep_amount) {
+        if($rep_amount >= 0) {
+            if($rep_amount >= 2000) {
+                return 'Legendary Shinobi';
+            }
+            elseif($rep_amount >= 1000) {
+                return 'Elite Shinobi';
+            }
+            elseif($rep_amount >= 600) {
+                return 'Respected Shinobi';
+            }
+            elseif($rep_amount >= 500) {
+                return 'Well-known Shinobi';
+            }
+            elseif($rep_amount >= 400) {
+                return 'Shinobi';
+            }
+            elseif($rep_amount >= 300) {
+                return 'Respected Villager';
+            }
+            elseif($rep_amount >= 200) {
+                return 'Well-known Villager';
+            }
+            elseif($rep_amount >= 100) {
+                return 'Villager';
+            }
+            else {
+                return 'New Villager';
+            }
```

Comment:
We reference these tiers elsewhere with hardcoded references to 1000 and 400 reputation - What if we want to change these values? 

Let's make this an array of tiers, and in e.g. the clan officer claim, check the amount and the name for e.g. `reputation_tier[8]` instead of `getRepName(1000)`.